rllib/algorithms/algorithm_config.py

```diff
@@ -765,6 +766,26 @@ def freeze(self) -> None:
     def validate(self) -> None:
         """Validates all values in this config."""
 
+        # TODO (sven): Add some structure to this method. Subdivide it into smaller
+        #  private methods, each responsible for one "theme" of the config, e.g.
+        #  evaluation, RLModule, env, etc...
+
+        # Check and error if `on_episode_created` callback has been overridden on the
+        # new API stack.
+        if self.uses_new_env_runners and self.callbacks_class is not DefaultCallbacks:
+            default_src = inspect.getsource(DefaultCallbacks.on_episode_created)
```

Comment:
We have this is_overriden pattern used across RLlib, can't we use that here instead of this inspect.getsource method?

Reply:
https://github.com/ray-project/ray/blob/7c818f2dd0603f083885d56810403a8d8a0d0151/rllib/utils/annotations.py#L167